src/main.ts

```diff
@@ -17,13 +17,15 @@ import { TokenService } from "services/TokenService";
 import { CeramicServiceMock } from "services/CeramicServiceMock";
 import { DealTokenSwap } from "entities/DealTokenSwap";
 import { IDataSourceDeals } from "services/DataSourceDealsTypes";
+import "./validation";
```

Comment:
do we need this line?

Reply:
Yes. Aurelia will read this file and register all the custom validation rules from it